src/lib/components/ScrollView/ScrollView.jsx

```diff
@@ -55,7 +57,7 @@ export const ScrollView = (props) => {
   const scrollPositionStart = direction === 'horizontal' ? 'left' : 'top';
   const scrollPositionEnd = direction === 'horizontal' ? 'right' : 'bottom';
   const scrollViewContentEl = useRef(null);
-  const scrollViewViewportEl = useRef(null);
+  const scrollViewViewportEl = ref ?? React.createRef(null);
```

Comment:
I am not sure, but I think there should be `useRef`. `useRef` remembers its state even if component is re-rendered, `React.createRef` just create reference every single pre-render.

Reply:
There is a problem with calling a `useRef` hook conditionally. I can get around it by introducing an auxiliary variable for the hook and using it in the condition instead. Is it the correct way to address this problem?